### PR TITLE
Compare route

### LINKED ISSUE #63: Complete Routes
# User Story
As a **programmer**,
I want to **connect my frontend to mongoDB**,
so I can **stable connection between front- & backend**.

# Description

#Material

# Done

- [x] Connect product routes
- [x] Connect supermarket routes
- [x] ...

### PR DESCRIPTION
**Pull request by LebkuchenCJ**
Fix #63 

## Deployment: 
(Click on SIGN IN and create a list and add products and compare those)
https://pryces-compare-route-f19liar5w.herokuapp.com/

## Comments: 
- Add final route for compare function

image

### CODE REVIEW

File: lib/models/supermarkets.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { getCollection } = require("../database");
+
+async function readSupermarkets() {
+  const supermarketCollection = await getCollection("supermarkets");
+  const supermarkets = await supermarketCollection.find({}).toArray();
+  if (!supermarkets) {
+    return null;
+  }
```
Comment (lmachens) [inline lib/models/supermarkets.js:8]: I remember to ask you this question before:
In which case is `supermarkets` falsy?
I don't think this case will ever happen here.

**Review by lmachens — COMMENTED** [review]
👍

File: lib/models/supermarkets.js (hunk 1 above)
Comment (LebkuchenCJ) [reply lib/models/supermarkets.js:8]: It should be zero, if someone would delete them from mongoDB, isn't it?

Comment (lmachens) [reply lib/models/supermarkets.js:8]: @JonasImm 
I think it should be an empty array (`[]`). 
There is not `zero` in JavaScript.

Is an empty array falsy or truthy?

**Pull request merged**